javaslang/src/main/java/javaslang/Value.java

```diff
@@ -1032,8 +1035,8 @@ default CharSeq toCharSeq() {
         final Comparator<T> comparator;
         if (this instanceof TreeSet<?>) {
             return (TreeSet<T>) this;
-        } else if (this instanceof SortedSet<?>) {
-            comparator = ((SortedSet<T>) this).comparator();
+        } else if (this instanceof Ordered<?>) {
```

Comment:
Don't know why we used the `<?>` everywhere in instanceof checks - they are not really needed and add no value here. Nevermind, you don't need to change all occurrences in this PR.